Clean up a bit of npm scripts

AlexKVal opened the pull request:
I'll remove `lib/generators/react_on_rails/templates/node/base/client/node/server.js`
after the master branch will be free of linting issues.

--
green https://travis-ci.org/AlexKVal/react_on_rails/builds/153960979

justin808 commented:
Some comments...

---

Reviewed 7 of 7 files at r1.
Review status: all files reviewed at latest revision, 3 unresolved discussions.

---

*[.eslintignore, line 14 \[r1\]](https://reviewable.io:443/reviews/shakacode/react_on_rails/531#-KPj2mtUixt6mjlO9i2z:-KPj2mtUixt6mjlO9i3-:borbpde) ([raw file](https://github.com/shakacode/react_on_rails/blob/fbd35a10f681c010b112591efdb8f9cbab6ce078/.eslintignore#L14)):*
> ```
> **/coverage/**
> 
> lib/generators/react_on_rails/templates/node/base/client/node/server.js
> ```

we're fixing this, so remove this line

---

*[package.json, line 52 \[r1\]](https://reviewable.io:443/reviews/shakacode/react_on_rails/531#-KPj2qSc57icFVrpzenK:-KPj2qSc57icFVrpzenL:b-e40jo2) ([raw file](https://github.com/shakacode/react_on_rails/blob/fbd35a10f681c010b112591efdb8f9cbab6ce078/package.json#L52)):*
> ```JSON
>     "prepublish": "npm run build",
>     "babel": "babel --out-dir node_package/lib node_package/src",
>     "build": "npm run clean && npm run babel",
> ```

The problem with this, is that we can't add comments easily.

If we duplicate, then it's not DRY.

Hmmm....

---

*[node_package/scripts/test, line 9 \[r1\]](https://reviewable.io:443/reviews/shakacode/react_on_rails/531#-KPj2xu5Az4M5Fs0T2ab:-KPj2xu5Az4M5Fs0T2ac:btzpjtd) ([raw file](https://github.com/shakacode/react_on_rails/blob/63aa9bc1192e150ca7c6bac9e0ef151cc6328479/node_package/scripts/test#L9)):*
> ```
> # If we want to run tests in the browser
> # echo "Running JS tests in browser"
> # $(npm bin)/browserify -t babelify node_package/tests/*.js | tape-run | $(npm bin)/faucet
> ```

These are useful comments, at least they used to be.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/shakacode/react_on_rails/531#-:-KPj3AoDEzInaB9ZhXOL:bn8acj7)*
<!-- Sent from Reviewable.io -->

AlexKVal commented:
Review status: 6 of 7 files reviewed at latest revision, 3 unresolved discussions, some commit checks broke.

---

*[package.json, line 52 \[r1\]](https://reviewable.io:443/reviews/shakacode/react_on_rails/531#-KPj2qSc57icFVrpzenK:-KPnDZsRtGdw78mN4oZX:byhbgtr) ([raw file](https://github.com/shakacode/react_on_rails/blob/fbd35a10f681c010b112591efdb8f9cbab6ce078/package.json#L52)):*
<details><summary><i>Previously, justin808 (Justin Gordon) wrote…</i></summary>
> The problem with this, is that we can't add comments easily.
> 
> If we duplicate, then it's not DRY.
> 
> Hmmm....
</details>
the point is
```diff
+ 6
− 39
```
lines of code.

---

*[node_package/scripts/test, line 9 \[r1\]](https://reviewable.io:443/reviews/shakacode/react_on_rails/531#-KPj2xu5Az4M5Fs0T2ab:-KPnDm5X4WYNE88JF2TA:bz69v3b) ([raw file](https://github.com/shakacode/react_on_rails/blob/63aa9bc1192e150ca7c6bac9e0ef151cc6328479/node_package/scripts/test#L9)):*
<details><summary><i>Previously, justin808 (Justin Gordon) wrote…</i></summary>
> These are useful comments, at least they used to be.
</details>
The better place for such kind of information are `Contributing.md` and `Readme.md` files.

I've added instructions to `Contributing.md` about running tests in a browser.
though this way tests throw some error.
![browserify.png](https://files.reviewableusercontent.io/images/reviewable/847572/-HCbanZtCluUHaVo8ug4KHtZ/browserify.png)

---

*[.eslintignore, line 14 \[r1\]](https://reviewable.io:443/reviews/shakacode/react_on_rails/531#-KPj2mtUixt6mjlO9i2z:-KPnFHalKYVqdm-ClaGl:b-al0way) ([raw file](https://github.com/shakacode/react_on_rails/blob/fbd35a10f681c010b112591efdb8f9cbab6ce078/.eslintignore#L14)):*
I've removed it but now it seems there is another error in the master branch.
> The command "rake examples" failed and exited with 1 during .

https://travis-ci.org/shakacode/react_on_rails/builds/154212067

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/shakacode/react_on_rails/531)*
<!-- Sent from Reviewable.io -->

justin808 commented:
Progress!

---

Reviewed 2 of 2 files at r2.
Review status: all files reviewed at latest revision, 2 unresolved discussions.

---

*[node_package/scripts/test, line 9 \[r1\]](https://reviewable.io:443/reviews/shakacode/react_on_rails/531#-KPj2xu5Az4M5Fs0T2ab:-KPqB0G5eCHQrRLTixGE:b-9fc7nd) ([raw file](https://github.com/shakacode/react_on_rails/blob/63aa9bc1192e150ca7c6bac9e0ef151cc6328479/node_package/scripts/test#L9)):*
<details><summary><i>Previously, AlexKVal (Alexander Shemetovsky) wrote…</i></summary>
> The better place for such kind of information are `Contributing.md` and `Readme.md` files.
> 
> I've added instructions to `Contributing.md` about running tests in a browser.
> though this way tests throw some error.
> ![browserify.png](https://files.reviewableusercontent.io/images/reviewable/847572/-HCbanZtCluUHaVo8ug4KHtZ/browserify.png)
> 
</details>
Hmmm...that's concerning, actually!

---

*[.eslintignore, line 14 \[r1\]](https://reviewable.io:443/reviews/shakacode/react_on_rails/531#-KPj2mtUixt6mjlO9i2z:-KPqB81mJ1gKKsoF5Hjg:bp1i748) ([raw file](https://github.com/shakacode/react_on_rails/blob/fbd35a10f681c010b112591efdb8f9cbab6ce078/.eslintignore#L14)):*
<details><summary><i>Previously, AlexKVal (Alexander Shemetovsky) wrote…</i></summary>
> I've removed it but now it seems there is another error in the master branch.
> > The command "rake examples" failed and exited with 1 during .
> 
> https://travis-ci.org/shakacode/react_on_rails/builds/154212067
</details>
Please take a look.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/shakacode/react_on_rails/531#-:-KPqBADmGXLvq4IcGR1-:b2m4ljj)*
<!-- Sent from Reviewable.io -->

AlexKVal commented:
Let's create a new issue and with it there. #533
This PR is just about npm-scripts.

justin808 commented:
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/>

---

Review status: all files reviewed at latest revision, 2 unresolved discussions.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/shakacode/react_on_rails/531#-:-KPqVqBXDFN_qzU7BNjx:bnfp4nl)*
<!-- Sent from Reviewable.io -->
